main.go

```diff
@@ -108,31 +109,79 @@ func main() {
 		setupLog.Error(err, "unable to start manager")
 		os.Exit(1)
 	}
+
+	certsReady := make(chan struct{})
+	if err = kueuev1alpha1.ManageCerts(mgr, certsReady); err != nil {
+		setupLog.Error(err, "unable to set up cert rotation")
+		os.Exit(1)
+	}
+
+	cCache := cache.New(mgr.GetClient())
+	queues := queue.NewManager(mgr.GetClient(), cCache)
+
+	if err := setupIndexes(mgr); err != nil {
+		setupLog.Error(err, "Unable to setup indexes")
+	}
+
+	setupProbeEndpoints(mgr, certsReady)
+	go startControllers(mgr, cCache, queues, certsReady, config.ManageJobsWithoutQueueName)
+
+	ctx := ctrl.SetupSignalHandler()
+	go func() {
+		queues.CleanUpOnContext(ctx)
+	}()
+
+	startScheduler(ctx, mgr, cCache, queues, certsReady)
+
+	setupLog.Info("starting manager")
+	if err := mgr.Start(ctx); err != nil {
+		setupLog.Error(err, "problem running manager")
+		os.Exit(1)
+	}
+}
+
+func setupIndexes(mgr ctrl.Manager) error {
 	if err := queue.SetupIndexes(mgr.GetFieldIndexer()); err != nil {
-		setupLog.Error(err, "Unable to setup queue indexes")
+		return err
```

Comment:
log here instead of returning, so that we can at least configure the other indexes.